docs/_ui.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import {Pre, Box, Border} from 'rebass'
+import {Pre} from 'rebass'
```

Comment:
Does this cause any errors? Can you still build the docs without problems?

Reply:
This parameter is an unused variable.
I think, even if it is used temporarily, the value will be provided below.

https://github.com/mdx-js/mdx/blob/022b2c6e7a0d200d6f5c8f57205e40ca1bdbbacf/docs/_app.js#L11

Reply:
This is a separate case, but if you run `yarn run docs: build` when testing travis-ci you will not have to always ask this question.